sys/linux/socket_netlink_generic_ethtool.txt.const

```diff
@@ -2,39 +2,13 @@
 arches = 386, amd64, arm, arm64, mips64le, ppc64le, riscv64, s390x
 ETHTOOL_A_BITSET_BITS = 3
 ETHTOOL_A_BITSET_BITS_BIT = 1
-ETHTOOL_A_BITSET_BIT_INDEX = 1
```

Comment:
I think this is fine.
Currently compiler associates each const with only 1 file, so these just moved to auto.txt.const.
I don't remember why I did it this way, and if it was intentional or not.
But either way it's not related to this change, and everything should work fine (we need each const in at least 1 const file).

Reply:
Thanks!